CourseDevelopment/Week8/Readme.MD

```diff
@@ -0,0 +1,194 @@
+# Course Development Week 8 The scientific process, beyond the PhD
+
+Course Development Week 8 for the Open Science Course for physics students at graduate level. This document is based on the Syllabus.MD.
+
+Date: Feb/28/2023
+
+Name: Hendrik Snijder @hendelhendel
+
+Open tasks @SanliFaez @erikvansebille
++ Feedback on lecture-tutorial distribution of topics
++ Feedback on lecture format
++ Feedback on tutorial format
+
+## Goal: open design and open hardware; prototyping? 
+
+Dalle2 & @hendelhendel: A grant proposal and money for science, while a duck is doing research in her lab.
+![eendjefunding](https://user-images.githubusercontent.com/93695286/222175974-54ffc694-d1b4-4e7b-bdd7-7f9d8c94f2d8.png)
+
+## Keywords student familiarity check week 8
+
+|**New Concepts**|**Familiar Concepts**|
+|----------------|---------------|
+|Careers in science and beyond | Mental health in academia|
+|Applying for scientific positions | Literature and referencing|
+|Data stewardship|CV and applications|
+|Grant writing|
+|the PhD advantage|
+|Preprinting and peer-review|
+|Altmetrics and visibility|
+|Recognition and rewards|
+|Science in transition|
+
+
+## Lecture: Week 8
+**Topics in lecture**
+Grant writing | Pre-printing & Peer-review | Altmetrics and visibility | Science in transition | literature and referencing. 
+
+### Lecture format 
+**Goal: After this lecture, students understand the scientific process**
+
+A short lecture divided in two parts covering the following topics: Grant writing, Pre-printing & Peer-review, Altmetrics and visibility, Science in transition, literature and referencing. 
+Each part answers a specific question, regarding a future in science.
+
+**Student preparation:**
++ **!Answer**: How would your work-life look like as a successful scientist in academia, expressed in **five?** keywords in an mindmap? (This mindmap will be used in the tutorial for grading).  See this link for background information about mindmaps: https://www.mindmapping.com/mind-map 
++ Read: https://elifesciences.org/articles/16800
++ Watch: https://www.youtube.com/watch?v=2EnvA4Wh6-8&t=5s 
++ Browse: https://www.academictransfer.com/en/ 
+
+
+**part 1:** How to share your knowledge and claim your place as a researcher in (Open) Science in terms of citations, media attention, potential collaborators, job opportunities and funding opportunities?
+
+The teacher will answer this question in a talk. At least the following topics are covered: Literature and referencing | Preprinting and peer-review | Altmetrics and visibility. See below for references.
+
+**Part 2:** How to fund a (future) scientific research project?
+The teacher will answer this question in a talk. Where at least the following topics are covered:
+Science in transition 
+DORA, 
+Grant writing (a hint to data stewardship covert by tutorial can be made, )
+How to write a successful grant application? 
+What is traditional
```

Comment:
What do you mean with traditional?

Reply:
I will make the change:
What is traditional What is expected to be new/ what will be the shift of focus in grant proposals in the (near) future? 
--> 
What is typically in a grant application now a days? Will there be a shift of focus in grant proposals in the (near) future?